frontend/src/lib/error-output.tsx

```diff
@@ -13,19 +13,10 @@ export function getOperatorError(configMaps: ConfigMap[], isOperatorInstalled: b
                 {openShiftConsoleUrl && openShiftConsoleUrl !== '' ? (
                     <div>
                         {t('Install the Operator through the following link: ')}
```

Comment:
Since we don't use `openShiftConsoleUrl` and in 2.7 we know we are running inside the OCP console, we should be able to remove this. Do we use the `console-public` ConfigMap anywhere else? If not, I think there might be a watch for it we can remove.